lib/core/src/Cardano/Wallet/Network.hs

```diff
@@ -176,7 +176,7 @@ waitForNetwork nw policy = do
 -- for no longer than a minute.
 defaultRetryPolicy :: Monad m => RetryPolicyM m
 defaultRetryPolicy =
-    limitRetriesByCumulativeDelay (60 * second) (exponentialBackoff 10000)
+    limitRetriesByCumulativeDelay (3600 * second) (exponentialBackoff 10000)
```

Comment:
this could be lowered (or removed), once we are updated to include https://github.com/input-output-hk/jormungandr/pull/977

Reply:
We are actually going to change the start-up logic. Right now, you can't start the wallet if there's no backend to connect to, but that's bonkers. The wallet can simply wait an poll regularly until a backend shows up; so this whole timeout will go away here.